paket.dependencies

```diff
@@ -1,11 +1,11 @@
-framework: netstandard2.0
+framework: netstandard2.0, net462, netcoreapp2.0
 redirects: force
 source https://api.nuget.org/v3/index.json
 
-nuget System.Runtime.Caching
-nuget System.Configuration.ConfigurationManager
+nuget System.Runtime.Caching prerelease
```

Comment:
added prereleases here because adding `netcoreapp2.0` as a framework brings along 4.5-prerelease versions of some libraries and these needed to be allowed for a successful resolution.